pylabrobot/liquid_handling/resources/tilt_module.py

```diff
@@ -13,7 +13,6 @@ def __init__(
     name: str,
     backend: TiltModuleBackend,
   ):
-    # TODO: Measure size of tilt module.
```

Comment:
I think this is specific to the tilt module you are using. e.g. i'm using hamilton, but others maybe using a different version.